lib/samples/add_integrated_mesh_layer/add_integrated_mesh_layer.dart

```diff
@@ -0,0 +1,93 @@
+// Copyright 2025 Esri
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+import 'package:arcgis_maps/arcgis_maps.dart';
+import 'package:arcgis_maps_sdk_flutter_samples/common/common.dart';
+import 'package:flutter/material.dart';
+
+class AddIntegratedMeshLayer extends StatefulWidget {
+  const AddIntegratedMeshLayer({super.key});
+
+  @override
+  State<AddIntegratedMeshLayer> createState() => _AddIntegratedMeshLayerState();
+}
+
+class _AddIntegratedMeshLayerState extends State<AddIntegratedMeshLayer>
+    with SampleStateSupport {
+  // Create a controller for the scene view.
+  final _sceneViewController = ArcGISSceneView.createController();
+
+  // Create an IntegratedMeshLayer with the URI to an integrated mesh layer scene service.
+  final _integratedMeshLayer = IntegratedMeshLayer.withUri(
+    Uri.parse(
+      'https://tiles.arcgis.com/tiles/z2tnIkrLQ2BRzr6P/arcgis/rest/services/Girona_Spain/SceneServer',
+    ),
+  );
+
+  // A flag for when the map view is ready and controls can be used.
+  var _ready = false;
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      body: Stack(
+        children: [
+          ArcGISSceneView(
+            controllerProvider: () => _sceneViewController,
+            onSceneViewReady: onSceneViewReady,
+          ),
+          // Display a progress indicator and prevent interaction until state is ready.
+          LoadingIndicator(visible: !_ready),
+        ],
+      ),
+    );
+  }
+
+  Future<void> onSceneViewReady() async {
+    // Create a scene.
+    final scene = await _setupScene();
+
+    // Add the layer to the scene's operational layers.
+    scene.operationalLayers.add(_integratedMeshLayer);
+    _sceneViewController.arcGISScene = scene;
+
+    setState(() => _ready = true);
+  }
+
+  Future<ArcGISScene> _setupScene() async {
+    // Create a scene.
+    final scene = ArcGISScene.withBasemapStyle(BasemapStyle.arcGISTopographic);
```

Comment:
We should use the imagery basemap to match .NET. The design isn't very clear here so I'll make a note to follow up on clarifying some of this with the samples design council.